ui/StatusQ/src/StatusQ/Layout/StatusAppNavBar.qml

```diff
@@ -42,7 +42,7 @@ Rectangle {
         id: layout
         anchors {
             fill: parent
-            topMargin: 48
+            topMargin: Qt.platform.os === "osx" ? 48 : 12 // space reserved for Mac traffic lights (window icons)
```

Comment:
```suggestion
            topMargin: Qt.platform.os === "osx" ? 48 : 12 // space reserved for Mac traffic lights (window icons)
```
```suggestion
            topMargin: Qt.platform.os === Constants.mac ? 48 : 12 // space reserved for Mac traffic lights (window icons)
```

Reply:
No can't do, this is StatusQ, those Constants are defined in status-desktop (one level up)